tests/unit/App.spec.ts

```diff
@@ -1695,3 +1777,273 @@ describe('App as an historical business', () => {
     expect(vm.$store.state.reasonText).toContain('December 6, 2021 at 12:05 pm Pacific time')
   })
 })
+
+describe('App as a Draft Registration with approved NR', () => {
```

Comment:
I suggest you create a new file with this content, `App2.spec.ts`, to avoid excessively long files. (Not blaming anyone here.)

Or `App-reg.spec.ts` ? Up to you.

Reply:
Sounds good, was thinking the same, will do 👍

Reply:
Updated